lighthouse-core/audits/autocomplete.js

```diff
@@ -43,30 +185,83 @@ class AutocompleteAudit extends Audit {
     };
   }
 
+  /**
+   * @param {LH.Artifacts.FormInput} input
+   * @return {{isValid: Boolean, orderWarn?: Boolean}}
+   */
+  static isValidAutocomplete(input) {
```

Comment:
i like that this method is broken out. nice.

tho i would expect the `is` prefix means the return type is boolean. (I suppose it was before you came across the order thing.. )

I agree in using the prop=attribute inference to check order (once all tokens are considered valid)... so i like having both signals in here...

so ... how about we call this method `checkAttributeValidity()`

and its two props in the return are `hasValidTokens` and `inValidOrder` (note that the boolean flips for inValidOrder)